packages/notifi-react/README.md

```diff
@@ -1,81 +1,139 @@
 # Notifi React SDK
 
-`@notifi-network/notifi-react` accelerates the development of Notifi integrated dapps by providing:
+`@notifi-network/notifi-react` accelerates the development of Notifi integrated dapps.
 
-- Out-of-the-box React components (`NotifiCardModal`) by which developers can just plug and play.
+-  [Out of The Box Implementation](#out-of-the-box-implementation)
+-  [Customized Implementation](#advanced-customized-implementation)
+-  [Creating Your Own Card](#creating-your-own-card)
```

Comment:
I would not say `Creating your Own Card`. 
Consuming `react context` is able to "Customize the dapp". It is necessarily  "Card". Could be anything

Reply:
ah fair,